src/main/java/at/ac/tuwien/kr/alpha/grounder/internalatoms/ChoiceAtom.java

```diff
@@ -1,8 +1,9 @@
-package at.ac.tuwien.kr.alpha.common.atoms;
+package at.ac.tuwien.kr.alpha.grounder.internalatoms;
```

Comment:
I very much like the separation. "internalatoms" is unusually long ... I think that "atoms" should suffice.